### PR TITLE
V8

### PR DESCRIPTION
**Pull request by gururajsh**
Thank you for contributing to the CF CLI! Please read the following:

* Please make sure you have implemented changes in line with the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/main/.github/CONTRIBUTING.md)
* We're not allowed to accept any PRs without a signed CLA, no matter how small.
If your contribution falls under a company CLA but your membership is not public, expect delays while we confirm.
* All new code requires tests to protect against regressions.
* Contributions must be made against the appropriate branch. See the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/main/.github/CONTRIBUTING.md)
* Contributions must conform to our [style guide](https://github.com/cloudfoundry/cli/wiki/CLI-Product-Specific-Style-Guide). Please reach out to us if you have questions.

## Does this PR modify CLI v6, CLI v7, or CLI v8?
CLI v8

Please see the contribution doc above or review [Architecture Guide](https://github.com/cloudfoundry/cli/wiki/Architecture-Guide).

## Description of the Change

Golang version upgrade to 1.19

## Why Is This PR Valuable?

What benefits will be realized by the code change? What users would want this change? What user need is this change addressing? 

## Why Should This Be In Core?

Explain why this functionality should be in the cf CLI, as opposed to a plugin. 

## Applicable Issues

List any applicable GitHub Issues here

## How Urgent Is The Change?

Is the change urgent? If so, explain why it is time-sensitive.

## Other Relevant Parties

Who else is affected by the change?

### CODE REVIEW

**Review by PeteLevineA — APPROVED** [review]
The time.sleeps in the tests are fine?

**Comment by gururajsh** [comment]
Yes, they are to reduce flakiness

File: integration/v7/isolated/scale_command_test.go [hunk 1]
```diff
@@ -300,6 +301,10 @@ var _ = Describe("scale command", func() {
 						buffer := NewBuffer()
 						_, err := buffer.Write([]byte("y\n"))
 						Expect(err).ToNot(HaveOccurred())
+
+						//Delay to reduce flakiness
+						time.Sleep(3 * time.Second)
```
Comment (a-b) [inline integration/v7/isolated/scale_command_test.go:306]: Per conversation with @gururajsh in order to address issues with flakey tests, we will be taking this as a tech debt.

File: integration/v7/push/instances_flag_test.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ var _ = Describe("push with instances flag", func() {
 				Eventually(session).Should(Exit(0))
 			})
 
-			time.Sleep(5 * time.Second)
+			time.Sleep(10 * time.Second)
```
Comment (a-b) [inline integration/v7/push/instances_flag_test.go:32]: @gururajsh please consider using a centralized sleep time constant to control timings across the whole test suit.

**Review by a-b — APPROVED** [review]
LOTM

**Pull request merged**